hal/src/generate/ResourceType.txt

```diff
@@ -83,3 +83,4 @@ kResourceType_PWMVictorSPX = 81
 kResourceType_RevSparkMaxPWM = 82
 kResourceType_RevSparkMaxCAN = 83
 kResourceType_ADIS16470 = 84
+kResourceType_DutyCycle = 85
```

Comment:
Nope.  This hasn't been assigned a resource type yet, and 85 is already allocated.  We should update this list to match the v5 list though (in a separate PR).  See https://github.com/wpilibsuite/ni-libraries/blob/master/src/include/FRC_NetworkCommunication/UsageReporting.h